content/docs/studio/user-guide/billing-and-payment.md

```diff
@@ -37,81 +40,76 @@ team. All invoice amounts are calculated for one billing period at a time. A
+added to your account for the remaining days in the billing period. You can
+apply for a future payment.
 
 When you downgrade from the Team plan to the Free plan, you will receive a
-refund based on the number of days remaining in the billing period. For any
-inquiries about refunds, please [contact us](mailto:support@iterative.ai).
+prorated refund for the days remaining in the billing period. For inquiries
+about refunds, please [contact us].
 
-To upgrade to or downgrade from an Enterprise plan, please
-[contact us](mailto:support@iterative.ai).
+To upgrade to or downgrade from an Enterprise plan, please [contact us].
 
-## Billing period, auto-renewal of team plan, and grace period for non-payment
+## Billing period and auto-renewal of Team plans
 
 The billing period for each team plan subscription starts on the date of the
-subscription, and is one month long. For example, if your subscription date is
+subscription and is one month long. For example, if your subscription date is
 the 15th of May, your next billing period will start on the 15th of June.
 
-All Team plan subscriptions are renewed automatically at the end of the billing
-period.
+Team plan subscriptions are renewed automatically for one more month at the end
+of each billing period.
 
-Seven (7) days before the end of your billing period (month), you will receive
-an email notification indicating that the end date is approaching. You should
-maintain a valid payment method so that we can process your payment and
-auto-renew your subscription at the end of the billing period.
+Seven (7) days before the end of your billing period you will receive an email
+notification indicating that the end date is approaching. You should maintain a
+valid payment method so that we can process your payment and auto-renew your
+subscription at the end of the billing period.
 
-At the end of your billing period, if we were unable to renew your payment, you
-will enter a grace period of 5 days, during which you must ensure payment.
-During the grace period, you will continue to be able to use Iterative Studio in
-full capacity. After the grace period expires, you and all your team members
-will lose access to your team's views. You will continue to have access to your
-team's Settings page. The access to the team's views will be restored once you
-clear the pending payment.
+## Grace period for non-payment
+
+At the end of your billing period, if we were unable to process your payment,
+you will enter a grace period of 5 days during which you must provide valid
+payment. During this time you will continue to have full access to Iterative
+Studio.
 
 Your account's primary email address will be notified after each failed payment.
-Payments may be retried up to 4 times within the next billing period. After
-this, if the payment does not succeed, you will lose access to your team's
-views. To restore the access, go to the `Plan and billing` section of your
```

Comment:
> After this, if the payment does not succeed, you will lose access to your team's views.

I removed this since it contradicts the grace period info provided next:

> After the grace period expires, you and all your team members will lose access...